src/main/java/dave/exceptions/InvalidCommandException.java

```diff
@@ -0,0 +1,8 @@
+package dave.exceptions;
```

Comment:
I like you idea of having different kinds of exception classes to handle different situations